lib/server/index.js

```diff
@@ -8,23 +8,33 @@ module.exports = function () {
 
   return function (env_config, callback) {
 
-    var pack = new Hapi.Pack();
+    var server = new Hapi.Server();
 
-    env_config.hooks.runStatic('server.pack.pre', [pack]);
+    env_config.hooks.runStatic('server.pack.pre', [server]);
 
-    pack.server(env_config.www_port, {
+    server.connection({
+      port: env_config.www_port,
       labels: ['web'],
-      cors: true,
-      payload: {
-        maxBytes: 1048576 * 10 // 10 MB
+      routes: {
+        cors: {
+          override: false
+        },
+        payload: {
+          maxBytes: 1048576 * 10 // 10 MB
+        }
       }
     });
 
-    pack.server(env_config.admin_port, {
+    server.connection({
+      port: env_config.admin_port,
       labels: ['admin'],
-      cors: true,
-      payload: {
-        maxBytes: 1048576 * 10 // 10 MB
+      routes: {
+        cors: {
+          override: false
+        },
+        payload: {
+          maxBytes: 1048576 * 10 // 10 MB
+        }
```

Comment:
Maybe consolidate this and the previous connection configuration into a single object, while we are in here.